### PR TITLE
Archive

### PR DESCRIPTION
**Pull request by Nishant-96**
This PR includes features of archive
- Creating archive note card
- Archive. unarchive a note.
- delete archived note.
- also fixed sidenav css issue and closing of priority, label modal on outside click

### CODE REVIEW

**Comment by Tarandeep-s1ngh** [comment]
.

**Comment by Tarandeep-s1ngh** [comment]
https://tenor.com/view/tatti-sir-bad-gif-10311036

File: src/components/card/archiveNoteCard.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import "./notecard.css";
+import UnarchiveIcon from "@mui/icons-material/Unarchive";
+import DeleteIcon from "@mui/icons-material/Delete";
+import {
+  deleteArchiveNoteHandler,
+  postRestoreArchiveHandler,
+} from "../../utils";
+import { useAuth } from "../../context/auth/auth-context";
+import { useData } from "../../context/data/data-context";
```
Comment (mohammadkhizerkhan) [inline src/components/card/archiveNoteCard.jsx:11]: use barrel export

File: src/utils/index.js [hunk 2]
```diff
@@ -1,2 +1,5 @@
 export { postNotesHandler } from "./services";
 export { postNoteEditHandler } from "./services";
+export { postArchiveNoteHandler } from "./services";
+export { postRestoreArchiveHandler } from "./services";
+export { deleteArchiveNoteHandler } from "./services";
```
Comment (mohammadkhizerkhan) [inline src/utils/index.js]: export all this functions in a single export

**Review by mohammadkhizerkhan — APPROVED** [review]

**Review by ragini07 — APPROVED** [review]
-functionality working as mentioned
-Clean UI
-easily understood code
-just use barrel export

File: src/utils/index.js (hunk 2 above)
Comment (Nishant-96) [reply src/utils/index.js]: Done

File: src/components/card/archiveNoteCard.jsx (hunk 1 above)
Comment (Nishant-96) [reply src/components/card/archiveNoteCard.jsx:11]: missed out on this one. thanks for pointing it out.

**Pull request merged**